ma-copy-artist-album-to-clipboard.user.js

```diff
@@ -41,15 +41,40 @@ const tmpl = `<div class="copy-artist-notification"></div>
 function renderButtons() {
     const artist = document.querySelector('h1.band_name a').textContent;
     const bandDisco = document.querySelectorAll("#band_disco a.other, #band_disco a.album,  #band_disco a.single,  #band_disco a.demo");
+    let cpArr = []
     bandDisco.forEach(link => {
         const cpStr = `${artist} - ${link.textContent}`;
+        cpArr.push(cpStr);
         const elements = generateTemplate(tmpl, cpStr);
         const insertPoint = link.parentNode;
         insertPoint.style.position = 'relative';
         elements.forEach(elem => {
             link.parentNode.prepend(elem);
         });
     });
+    const cpStrs = cpArr.join("\n")
+    generateCopyAllButton(tmpl, cpStrs)
+}
+
+function generateCopyAllButton(html, cpStrs) {
```

Comment:
It would probably be good to reuse the generateTemplate function, but that can be fixed later.

Reply:
Yeah with some flags for sure